pagopa/pn-frontend
# feat(pn-3806): add participating entities page

## Short description
add participating entities page which shows a dynamic list generated by onetrust script for pf

## List of changes proposed in this pull request
- add ParticipatingEntities page for pf
- load onetrust script inside ParticipatingEntities page
- add /informativa-aderenti route

## How to test
update env file following the specifics available on confluence (a new env variable has been added for this feature), then run personafisica-webapp and go to "/informativa-aderenti" route. Verify the correct content is shown both with and without a logged user (public access)
